File: UNIT 1/py/sisi_miring.py
import math

# Input sisi a
sisi_a = float(input("Masukkan panjang sisi a (dalam satuan yang sama): "))

# Input sisi b
sisi_b = float(input("Masukkan panjang sisi b (dalam satuan yang sama): "))

# Hitung sisi miring dengan Teorema Pythagoras
sisi_miring = math.sqrt((sisi_a * sisi_a) + (sisi_b * sisi_b))

# Output hasil dengan 2 angka di belakang koma
print("Panjang sisi miring adalah: {:.2f}".format(sisi_miring))

File: UNIT 1/C/sisi_miring.c
#include <stdio.h>
#include <math.h>

int main() {
    float sisi_a, sisi_b, sisi_miring;
    printf("Masukkan panjang sisi a (dalam satuan yang sama): ");
    scanf("%f", &sisi_a);
    printf("Masukkan panjang sisi b (dalam satuan yang sama): ");
    scanf("%f", &sisi_b);
    sisi_miring = sqrt((sisi_a * sisi_a) + (sisi_b * sisi_b));
    printf("Panjang sisi miring adalah: %.2f\n", sisi_miring);
    return 0;
}